app/src/main/res/values/strings.xml

```diff
@@ -842,6 +842,10 @@ Before uploading your changes, the app checks with a &lt;a href=\"https://www.we
     <string name="quest_camera_type_fixed">Fixed</string>
     <string name="quest_camera_type_panning">Panning</string>
 
+    <string name="quest_camp_drinking_water_title">"Does it have drinking water?"</string>
+    <string name="quest_camp_shower_title">"Does it have showers?"</string>
+    <string name="quest_camp_power_supply_title">"Can you charge your phone/laptop here?"</string>
```

Comment:
what if only camp residents can charge? 

what if only some camp residents can charge? Say, only one with vans.

what if it is technically possible but extremely inaccessible (single connector in a bathroom, intended to be used for hair dryer)

what if you can charge phone (USB connector) but not laptop?

Reply:
I'm not quite sure what you mean by `what if only camp residents can charge`? Surely it is implied that only people having acquired legal access to camp may use its facilities (including power sockets)?

Perhaps `Can anyone charge their phone/laptop here?` would be better question? Or `Can any customer charge their phone/laptop here?` ?

As for the others, I have charged my phone in exactly such low-accessible places. Those are camps, not private hotel rooms - inconvenience with access to facilities - including power - is implied (e.g. in most smaller camps it is only one power socket / strip in common room, or socket in bathroom. That's why I always carry my own power strip, and multi-port USB charger to avoid fighting for scarce resources. Makes extra friends of the trip too :smile:). 
And if there is such strange situation (only DC USB port, no AC sockets), I'd expect user to leave a note (I've never encountered such situation myself though).

Reply:
I encountered case where there was one working socket, in bathroom stall (not very practical for charging :) )

+1 to `Can any customer charge their phone/laptop here?`

Reply:
Applied `Can any customer charge their phone/laptop here?`

Reply:
> I'm not quite sure what you mean by `what if only camp residents can charge`? Surely it is implied that only people having acquired legal access to camp (i.e. _"camp residents"_?) may use its facilities (including power sockets)?

In the UK at least it's quite common for there to be hookups for caravans (on blue Ceeforms), you may need to pay for those or they are only for caravan spots. It used to be quite common for there to only be those or a shaver socket in the bathroom, with no sensible ability to charge a phone/laptop via a domestic socket. I think that's becoming less common now because most mobiles need charging daily.

The detail on the wiki is pretty vague:
https://wiki.openstreetmap.org/wiki/Key:power_supply
https://wiki.openstreetmap.org/wiki/Tag:tourism%3Dcamp_site#Tagging

But I'm sure both caravan users and cyclists without an adaptor lead will be frustrated if they turn up to a site and find they can't plug in as someone has tagged it in the opposite way to what they were expecting despite it answering the SC quest question.